rita/src/client.rs

```diff
@@ -166,33 +166,33 @@ fn main() {
         env!("CARGO_PKG_VERSION"),
         env!("GIT_HASH")
     );
+
+    #[cfg(feature = "guac")]
+    {
+        SETTING.get_payment_mut().eth_address = guac_actix::CRYPTO.own_eth_addr();
+    }
+
     trace!("Starting with Identity: {:?}", SETTING.get_identity());
 
     let system = actix::System::new(format!("main {}", SETTING.get_network().own_ip));
 
     assert!(rita_common::debt_keeper::DebtKeeper::from_registry().connected());
-    assert!(rita_common::payment_controller::PaymentController::from_registry().connected());
     assert!(rita_common::tunnel_manager::TunnelManager::from_registry().connected());
     assert!(rita_common::http_client::HTTPClient::from_registry().connected());
     assert!(rita_common::traffic_watcher::TrafficWatcher::from_registry().connected());
+
     assert!(rita_client::exit_manager::ExitManager::from_registry().connected());
 
+    #[cfg(feature = "guac")]
```

Comment:
Maybe its a matter of taste but isn't this more readable for conditional compiling of code?

```rust
if cfg!(feature = "guac") {
  // foo
}
```
?

Reply:
The difference is that this way its cut out at compile time instead of at runtime